tests/integration/cli/projects/test/tests/test_test.py

```diff
@@ -1,2 +1,31 @@
+import pytest
```

Comment:
Our formatters require module-level definitions to be separated by two newline characters

Reply:
ahhh didn't spot the pre-commit <3